docusaurus/docs/create-a-plugin/extend-a-plugin/fetch-data-from-frontend.md

```diff
@@ -0,0 +1,287 @@
+---
+id: fetch-data-from-frontend
+title: Fetching data from frontend code
+description: Learn how to use the data proxy API to fetch data from frontend code in data source and app plugins in Grafana and overcome CORS limitations
+keywords:
+  - grafana
+  - plugins
+  - data proxy
+  - frontend
+  - data source
+  - CORS
+---
+
+# Fetching data from frontend data source and app plugins
+
+To fetch data from your data source an app plugins you have two options:
+
+- Use the browser [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) as you would do for any other javascript project
```

Comment:
Wouldn't you still use fetch even if hitting the Grafana API? Did you intend explaining browser connects directly to service vs grafana api?

Reply:
No. When you use the data proxy you use fetch from getBackendSrv. This fetch is the window,fetch from the broowser.